AWSAuthSDK/Sources/AWSUserPoolsSignIn/UserPoolsUI/AWSUserPoolForgotPasswordViewController.m

```diff
@@ -68,13 +68,17 @@ - (void)setUp {
     [self.tableView reloadData];
     [AWSUserPoolsUIHelper setUpFormShadowForView:self.tableFormView];
     [self setUpBackground];
+    
+    // setup button background
```

Comment:
As above, I'm confused by the use of a `tint` method to operate on a `background` property